go/vt/worker/worker.go

```diff
@@ -59,52 +59,90 @@ var (
 
 	statsState = stats.NewString("WorkerState")
 	// statsRetryCount is the total number of times a query to vttablet had to be retried.
-	statsRetryCount = stats.NewInt("WorkerRetryCount")
+	statsRetryCount = stats.NewCounter("WorkerRetryCount", "Total number of times a query to a vttablet had to be retried")
 	// statsRetryCount groups the number of retries by category e.g. "TimeoutError" or "Readonly".
-	statsRetryCounters = stats.NewCounters("WorkerRetryCounters")
+	statsRetryCounters = stats.NewCountersWithLabels("WorkerRetryCounters", "Number of retries", "category")
 	// statsThrottledCounters is the number of times a write has been throttled,
 	// grouped by (keyspace, shard, threadID). Mainly used for testing.
 	// If throttling is enabled, this should always be non-zero for all threads.
-	statsThrottledCounters = stats.NewMultiCounters("WorkerThrottledCounters", []string{"Keyspace", "ShardName", "ThreadId"})
+	statsThrottledCounters = stats.NewCountersWithMultiLabels(
+		"WorkerThrottledCounters",
+		"Number of times a write has been throttled",
+		[]string{"Keyspace", "ShardName", "ThreadId"})
 	// statsStateDurations tracks for each state how much time was spent in it. Mainly used for testing.
-	statsStateDurationsNs = stats.NewCounters("WorkerStateDurations")
+	statsStateDurationsNs = stats.NewGaugesWithLabels("WorkerStateDurations", "How much time was spent in each state", "state")
```

Comment:
Why not a Timer?

Reply:
Hmm. It's a good idea. Followup PR? I'd like to avoid breaking more in this one. : )